packages/datepickers/src/styled/StyledHeaderPaddle.ts

```diff
@@ -45,25 +71,19 @@ const COMPONENT_ID = 'datepickers.header_paddle';
 export const StyledHeaderPaddle = styled.div.attrs({
   'data-garden-id': COMPONENT_ID
 })<{
-  isCompact: boolean;
-  isHidden?: boolean;
+  $isCompact: boolean;
+  $isHidden?: boolean;
```

Comment:
This inherited code feels a bit lazy. Can we declare a `IStyledHeaderPaddle` interface, consistent with other view components?